pkg/util/containers/metrics/system/collector_linux.go

```diff
@@ -119,6 +120,7 @@ func newSystemCollector(cache *provider.Cache) (provider.CollectorMetadata, erro
 			collectors.OpenFilesCount = provider.MakeRef[provider.ContainerOpenFilesCountGetter](systemCollector, collectorLowPriority)
 			collectors.PIDs = provider.MakeRef[provider.ContainerPIDsGetter](systemCollector, collectorLowPriority)
 			collectors.ContainerIDForPID = provider.MakeRef[provider.ContainerIDForPIDRetriever](systemCollector, collectorLowPriority)
+			collectors.ContainerIDForInode = provider.MakeRef[provider.ContainerIDForInodeRetriever](systemCollector, collectorLowPriority)
```

Comment:
It won't work in this situation, you'd need host cgroup namespace access, which we don't have.

Reply:
Fixed 4ccc1a52b6339017629d344e27e05c9d79818950